.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     es2021: true,
   },
   extends: [
+    'plugin:jest-dom/recommended',
```

Comment:
Just a few preferences in the tests.. https://github.com/testing-library/eslint-plugin-jest-dom#recommended-configuration